pynobo.py

```diff
@@ -849,6 +849,31 @@ async def async_update_zone(self, zone_id, name=None, week_profile_id=None, temp
 
         await self.async_send_command(command)
 
+    def update_week_profile(self, week_profile_id, name=None, profile=None):
```

Comment:
Do we need the synchronous version?

Reply:
I think we can avoid adding synchronous wrappers for new methods. The existing wrappers were added for backwards compatibility.

Reply:
it is now removed